runtimes/common/src/constants.rs

```diff
@@ -408,6 +408,18 @@ pub mod preimage {
 	}
 }
 
+pub mod tips {
+	use super::*;
+
+	parameter_types! {
+		pub const DataDepositPerByte: Balance = deposit(0, 1);
```

Comment:
In Polkadot, this is set to 1 cent. However, I wanted to align with our other deposits.

Reply:
We already have a constant `ByteDeposit`. Maybe we use this across all pallets that require a byte deposit? I'm not sure if there are cases where we want to have different byte deposits for different pallets.

Reply:
Fixed in [96bb827](https://github.com/KILTprotocol/mashnet-node/pull/352/commits/96bb827aef6f6191a0bfa9747ebb02cdc0ed0833)